client/plots/survival.js

```diff
@@ -148,6 +148,13 @@ class TdbSurvival {
 								{ label: 'Tick', value: 'vtick' }
 							]
 						},
+						{
+							label: 'Survival Time Cut-Off',
+							type: 'number',
+							chartType: 'survival',
+							settingsKey: 'maxSurTime',
```

Comment:
@congyu-lu I prefer the config/setting key name to be `maxTimeToEvent`, since `maxSurTime` can mean only those who did not have an exit event. But this preference is not critical.

Reply:
thanks this is a good point, just changed it